Added generate kubectl token page (#4581)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/security/generate-token/index.md

---
layout: layout.pug
navigationTitle: Generate a Client Access Token
title: Generate a Client Access Token
menuWeight: 40
excerpt: How to generate a client access token
beta: false
enterprise: false
---

Obtain a token from the Kommander dashboard, on the user profile dropdown menu.

> ![Kommander-dashboard](./kommander-generate-token.png)

Reviewer: Maybe we should put this image after step 1.  Select the **Generate Token** button?

> 1.  Select the “Generate Token” button.

Reviewer: ```suggestion
1.  Select the **Generate Token** button, from the user profile dropdown in the upper right corner.
```

> 1.  Select an identity provider and cluster. If required, you will be prompted to login again.

Reviewer: ```suggestion
1.  Select an identity provider and cluster. If required, you will be prompted to log in again.
```

> 1.  Once logged-in, a page will show the commands required to configure kubectl to access the Konvoy cluster.

Reviewer: ```suggestion
1.  A page shows the commands required to configure `kubectl` to access your cluster.
```

> <p class="message--note"><strong>NOTE:</strong> When the token expires, it is necessary to repeat the above process to obtain a fresh token. When refreshing a token, only the <code>kubectl config set-credentials</code> command needs to be executed with the new token.</p>

Reviewer: ```suggestion
<p class="message--note"><strong>NOTE:</strong> When the token expires, you have to repeat the previous process to obtain a new token. When refreshing a token, only the <code>kubectl config set-credentials</code> command has to be executed with the new token.</p>
```
